modules/core/migrations/20240602202521-initial.sql

```diff
@@ -3,12 +3,12 @@ BEGIN;
 
 CREATE TABLE uploads
 (
-    id         SERIAL PRIMARY KEY,
-    name       VARCHAR(255) NOT NULL,
-    url        VARCHAR(255) NOT NULL,
---     uploader_id INT          REFERENCES users (id) ON DELETE SET NULL,
-    mimetype   VARCHAR(255) NOT NULL,
-    size       INT          NOT NULL,
+    id         VARCHAR(255) PRIMARY KEY,
+    url        VARCHAR(1024) NOT NULL   DEFAULT '',
+    name       VARCHAR(255)  NOT NULL   DEFAULT '',
```

Comment:
Also it's better not to embed defaults on the DB level, but rather application layer. Makes it more flexible. And DDD suggest to do that...

Reply:
If you have defaults at the application layer as well, then it's ok